pkg/provisioningv2/rke2/planner/planner.go

```diff
@@ -110,18 +116,19 @@ type roleFilter func(*planEntry) bool
 type Planner struct {
 	ctx                           context.Context
 	store                         *PlanStore
-	rkeControlPlanes              rkecontrollers.RKEControlPlaneClient
+	rkeControlPlanes              rkecontrollers.RKEControlPlaneController
```

Comment:
Out of curiosity, why are we changing to the `RKEControlPlaneController` here instead of the `RKEControlPlaneClient`?

Reply:
We can only enqueue with the controller, (which also is a client).